nodejs/week1/src/backend/routes/reservation.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const dataAccess = require("../dataAccess.js");
+const router = express.Router();
+const reservations = "/data/reservations.json";
```

Comment:
Remove. Same explanation than the previous file. Also export only router function. Import json using require directly.